package.json

```diff
@@ -53,16 +63,18 @@
     ]
   },
   "browserslist": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
+    "last 1 chrome version",
+    "last 1 firefox version",
+    "last 1 safari version"
   ],
   "devDependencies": {
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.16.7",
     "@babel/plugin-proposal-optional-chaining": "^7.16.7",
-    "@craco/craco": "^6.4.3",
+    "@babel/plugin-transform-async-to-generator": "^7.18.6",
+    "@craco/craco": "^6.4.4",
```

Comment:
Why is craco necessary?

Reply:
tTo enable configuration of the underlying webpack. Not this diff is not an inclusion of craco, but a version bump.